### PR TITLE
Add-remove setting

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

- Removed all hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task.
- Implemented a function for deleting a task.
- Implement a function for editing task descriptions.
- All changes to the To Do List are saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
+function saveToLocalStorage(data) {
+  tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  tasks.push(data);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
 
-const taskShelf = document.querySelector('#to-do-list__shelf');
+input.addEventListener('keyup', (event) => {
+  if (event.key === 'Enter') {
+    const inputValue = input.value;
```
Comment (joel-kalema) [inline src/index.js:18]: - [ ] Good job implementing a function for editing task descriptions 👍 , but currently we can't edit two descriptions and save them at the same time in local storage:
   - let's add two tasks `js` and `html`
   
Screenshot 2023-06-21 203740

   -  Then let edit them to `javaScript` and `HTML5`

Screenshot 2023-06-21 204129

  - Then let's refresh the page to see if we are saving the changes in the local storage 

Screenshot 2023-06-21 204417

As you can see, only the last task has been saved `HTML5` 😃. Kindly, fix it 👍

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Bleu-Yves-Sopoude 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

TN6


## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 


## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (DeliceLydia) [reply src/index.js:18]: Addressed

**Review by DeliceLydia — APPROVED** [review]
## Hi @Bleu-Yves-Sopoude, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed all changes requested


### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**